builder/qemu/builder.go

```diff
@@ -22,10 +21,10 @@ import (
 const BuilderId = "transcend.qemu"
 
 var accels = map[string]struct{}{
-	"none": struct{}{},
-	"kvm":  struct{}{},
-	"tcg":  struct{}{},
-	"xen":  struct{}{},
+	"none": {},
+	"kvm":  {},
+	"tcg":  {},
+	"xen":  {},
```

Comment:
Thanks for catching. I think this and the other one was due to an overly aggressive formatting plugin I was trying out. I'll revert the BC changes and re-push.